iap/generate_self_signed_jwt.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# Copyright 2024 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import google.auth
+import time
+import jwt
+import json
+
+from google.cloud import iam_credentials_v1
+
+def generate_jwt_payload(service_account_email: str, resource_url: str) -> str:
+  """Generates JWT payload for service account
+  
+  The resource url provided must be the same as the url of the IAP secured resource
```

Comment:
Please use full sentences for providing docstrings.

As well, please add docstrings for `service_account_email`, `resource_url` and the return.
```suggestion
  """Generates JWT payload for service account.
  
  The resource url provided must be the same as the url of the IAP secured resource.
```